app/controllers/assignments_controller.rb

```diff
@@ -118,6 +123,7 @@ def update
     unless params.key?(:assignment_form)
       @assignment = Assignment.find(params[:id])
       @assignment.course_id = params[:course_id]
+      puts "Perry1"
```

Comment:
Do not write to stdout.

Reply:
use "logger.debug" instead. and print messages that make sense e.g., @assignment.course_id instead of senseless strings.